sql/mob_groups.sql

```diff
@@ -4551,7 +4551,7 @@ INSERT INTO `mob_groups` VALUES (4515,776,92,0,128,0,0,0,75,80,0);
 INSERT INTO `mob_groups` VALUES (4516,889,92,0,128,0,0,0,60,60,0);
 INSERT INTO `mob_groups` VALUES (4517,896,92,1,0,0,0,0,1,1,0);
 INSERT INTO `mob_groups` VALUES (4518,903,92,1,0,0,0,0,1,1,0);
-INSERT INTO `mob_groups` VALUES (4519,928,92,21600,0,0,0,0,80,81,0);
+INSERT INTO `mob_groups` VALUES (4519,928,92,0,128,0,0,0,80,81,0); -- changed to lottery pop
```

Comment:
make sure the WotG version isn't using the same group. They need to be split up if so.